README.md

```
@@ -56,21 +56,43 @@
 
 <div align="center">
 
-### 🔥 Meet Team - Earendel
-
-  <ul>
-  <li><b>Rakibul Hasan</b> <i>(Team Leader)</i></li>
-  <li><b>Nafiul Hasan Hamim</b> </li>
-  <li><b>MFR Siam</b> </li>
-  <li><b>Mahmudul Hasan</b> <i>(Mentor)</i></li>
-  </ul>
+  <h2>🔥 Meet Team - <b>Earendel</b></h2>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: h2

(MD033, no-inline-html)